cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -1222,9 +1156,13 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size_t, 8)
           update_validity_and_row_indices_lists<decode_block_size_t, false, level_t>(
             processed_count, s, sb, nullptr, rep, t);
       } else {
+        // direct copy: no nulls, no lists, no need to update validity or row indices
+        // This ASSUMES that s->row_index_lower_bound is always -1!
```

Comment:
I think this comment isn't really needed here since we aren't using `s->row_index_lower_bound` here